borderless-explainer.md

```diff
@@ -344,7 +265,6 @@ find the IWA related implementation details from IWA related documents.
 ```
 {
   "name": "Borderless title bar example",
-  "display": "standalone",
   "display_override": ["borderless"]
```

Comment:
I see you removed `display: standalone`, but that is still required. User agents that do not understand `borderless` need to know what to fall back to. (PWAs should always provide `display` because by definition all the values in `display_override` are optional for UAs to implement.)